Tests/biddingTests.js

```diff
@@ -0,0 +1,94 @@
+var homePage = require('../Pages/homePage.js'),
+    loginPage = require('../Pages/loginPage'),
+    fashionCategoryPage = require ("../Pages/fashionCategoryPage"),
+    itemPage = require ("../Pages/itemPage"),
+    data = require("../Data/data.js");
+browser.waitForAngularEnabled(false);
+browser.ignoreSynchronization = true;
+browser.manage().window().maximize();
+
+describe("004: Bidding", function(){
+    beforeEach(()=> {
+        homePage.openPageURL(data.homepageLink)
+            .then(() => homePage.clickOnLogin())
+            .then(() => loginPage.enterEmail(data.userRafaNadal))
+            .then(() => loginPage.enterPassword(data.passwordUserRafaNadal))
+            .then(() => loginPage.clickOnLoginButton())
+            .then(() => homePage.waitForCategories())
+    })
+    afterEach(() =>{
+        homePage.clickOnLogoutButton();
+    })
+     it ("001: User is able to make a bid with decimal values", function (){
```

Comment:
Fix identation
<img width="736" alt="Screenshot 2020-11-23 at 11 18 17" src="https://user-images.githubusercontent.com/50828644/99951027-b8d41200-2d7d-11eb-9590-d635703436e8.png">

Reply:
I'll pay more attention, this happens when I comment out certain test case, and */ sign indents this. I fixed it.